### PR TITLE
feat: added button component

### PR DESCRIPTION
**Pull request by ultronjs**
Addition of Card Component and making sidebar card link active for all pages

### CODE REVIEW

File: css/components/card.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    justify-content: center;
+    align-items: center;
+    gap: 0.5rem;
+}
+
+.btn_card{
+    border: none;
+    background-color: transparent;
+    font-weight:500;
+    color: var(--primary-700);;
```
Comment (JantuDeb) [inline css/components/card.css]: remove the extra semicolon from the end of the line.

File: css/components/card.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    padding: 0.625rem;
+}
+.btn_card:hover{
+    background-color: var(--primary-700);
+    color:var(--white);
+    border-radius: 5px;
+    padding: 0.625rem;
+}
```
Comment (JantuDeb) [inline css/components/card.css]: You can reuse the button you have already created for component library

**Review by JantuDeb — CHANGES_REQUESTED** [review]
I have suggested some changes.

File: css/pages/card/card.html [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+<body>
+    <nav class="nav_bar">
+        <div class="nav_left_side">
+            <img class="nav_logo" src="../../../resources/logo.png" alt="logo">
+            <span><a class="heading_link" href="google.com">dashcss</a></span>
+        </div>
+        <div class="nav_right_side">
+            <span><a class="heading_link" href="google.com">Docs</a></span>
+            <span><a class="heading_link" href="google.com">Blog</a></span>
+            <span><a class="heading_link" href="google.com"><i class="header_icon link fa fa-github fa-lg">
```
Comment (tusharkandpal) [inline css/pages/card/card.html]: href attribute link ?

File: css/pages/card/card.html [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                    <iframe src="https://www.thiscodeworks.com/embed/61fa9e08b783be0015bbaee4"
+                        style="width: 100%; height: 463px;" frameborder="0">
+                    </iframe>
+                </div>
+            </div>
+        </div>
+        <div id="card_horizontal">
+            <h3 class="main_container_heading fs-m py-x-small">Horizontal Card</h3>
+            <p class="main_container_description py-x-small">Horizontal Card are those card which has width greater than its height.
+                Below is the example of the Horizontal Card.This card does not contain can space for any content.
```
Comment (tusharkandpal) [inline css/pages/card/card.html:139]: typo: add space after dot(.)

File: css/pages/card/card.html [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+                                    By Swapnil Bansal
+                                </div>
+                            </div>
+                            <div class="card_content_body">
+                                Leh is the joint capital and largest town of the union territory of Ladakh in India.
+                                Leh, located in the
+                                Leh district, was also the historical capital of the Kingdom of Ladakh,
+                            </div>
+                        </div>
+                        <div class="card_actions card_onlyIcon">
```
Comment (tusharkandpal) [inline css/pages/card/card.html]: be consistent with naming of classes i.e onlyIcon can be changed to only-icon

**Review by tusharkandpal — CHANGES_REQUESTED** [review]

File: css/pages/card/card.html (hunk 3 above)
Comment (ultronjs) [reply css/pages/card/card.html]: Dummy link.No issue , changed to the right ones for defined ones and the ones which are defined till now I have put #

File: css/components/card.css (hunk 1 above)
Comment (ultronjs) [reply css/components/card.css]: done

File: css/pages/card/card.html (hunk 5 above)
Comment (ultronjs) [reply css/pages/card/card.html]: Thanks removed the class.Not required

File: css/pages/card/card.html (hunk 4 above)
Comment (ultronjs) [reply css/pages/card/card.html:139]: it is because of formatting.Not effecting the output

**Review by tusharkandpal — APPROVED** [review]
good to go! can merge now

File: css/components/card.css (hunk 2 above)
Comment (ultronjs) [reply css/components/card.css]: Done

**Pull request merged**